_posts/2025-10-03-demystifying-datamovers.md

```diff
@@ -0,0 +1,93 @@
+---
+author: mattslotten
+date: 2025-10-03 16:00:09 -0400
+description: "In this blog post, we dive in on Kasten's data mover and backup target sizing"
+featured: false
+image: "/images/posts/2025-10-03-demystifying-datamovers/odyssey4.jpg"
+image_caption: ""
+layout: post
+published: true
+tags: [Sizing,Object Storage]
+title: "Demystifying Kasten's Datamover Repository Sizing"
+---
+Data movers are akin to a "secret sauce" when it comes to data protection solutions. They are essential to performing backup and restore operations and can be designed in a myriad of ways to optimize performance, scale, storage utilization, and security.  In the case of k8s application data protection solutions, we often see "off-the-shelf" Kopia in both opensource (e.g. Velero) and proprietary solutions (e.g. OADP, Dell PowerProtect), while other solutions either custom-build their own, or optimize existing data mover solutions (Kasten falls in the last two camps).
```

Comment:
> Kasten falls in the last two camps
I suppose you mean Kasten both use Kopia and developed it ? Or developed its own branch ? Or .. This "last two camps" were not clear for me but I'm probably a biased reader.